{
    "checksum": "643d3b23cb8c95f54fbd828f33fa6e2705da93da21462b5bd26133c16e865f54",
    "file": "troubleshooting.rst",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/plain"
    },
    "number_of_duplicates": 2,
    "package": "python-hpilo",
    "path": "python-hpilo/4.3-3/docs/troubleshooting.rst",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/python-hpilo/4.3-3/",
        "metric": {
            "size": 984
        },
        "pts_link": "https://tracker.debian.org/pkg/python-hpilo",
        "sloc": [
            [
                "python",
                2528
            ],
            [
                "ruby",
                467
            ],
            [
                "makefile",
                116
            ]
        ],
        "suites": [
            "bookworm",
            "bullseye",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/python-team/modules/python-hpilo",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/p/python-hpilo/4.3-3/docs/troubleshooting.rst",
    "stat": {
        "perms": "rw-r--r--",
        "size": 4862,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "4.3-3"
}